programs/clearing_house/src/context.rs

```diff
@@ -691,7 +691,6 @@ pub struct RepegCurve<'info> {
 pub struct MoveAMMPrice<'info> {
     #[account(
         has_one = admin,
-        constraint = state.admin_controls_prices
```

Comment:
why you get rid of this?

Reply:
if we want to keep it as a constraint was thinking to use exchange status instead, also isnt super relevant for v2